templates/collections.html

```diff
@@ -1,8 +1,20 @@
 {% extends "_base.html" %} {% block body %}
 <div class="container">
   <div class="full-width-bg component">
-    <p>Under construction. Want to help out? Visit:
-      <a href="https://github.com/ansible/jinja-docsite" target="_blank">Ansible community docsite prototype</a>
+    <p>
+      The Ansible community documentation site has recently changed.
+      The content on this page has been removed.
+    </p>
+    <p>
+      Here are some links you might find useful instead:
+      <ul>
+        <li>
+          <a href="{{ pages.site.index }}">Ansible community documentation home page</a>
+        </li>
+        <li>
+          <a href="{{ links.pkg_docs.index }}">Ansible core documentation</a>
+        </li>
```

Comment:
Seems this one should link to the collections index, not the main core or community docs. But couldn't we do this with an http redirect in ansible/docsite instead?

Reply:
doh. fixed the link.

we could add a redirect but a stub page might get us more feedback after we go live. I'm going to add feedback links in a separate PR.